content/en/Getting started/customize-report.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Customize Your Pentest Report"
+linkTitle: "Customize Your Report"
+weight: 310
+description: >
+  You may be able to customize your pentest report.
```

Comment:
Main thing I noticed might be a clarification to make: You can't customize the Full/Full+Findings/Customer Letter/Attestation reports. When you customize a report, it appears as a separate item at the bottom of the drop-down.

The other formats are preset formats. Including the "Customized Report" option at the bottom of the list, and mentioning that it's either one of the preset formats or a custom report might fix that.

![Screen Shot 2022-04-29 at 10 53 51 AM](https://user-images.githubusercontent.com/5684683/165969470-aa39fa70-6155-4ca7-a522-5cd7835d3c69.png)